Assets/Scripts/Audio/AddThisToEverytScriptThatCallsAudio.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(AudioManagerCombat))]
+[RequireComponent(typeof(AudioManagerMovement))]
+[RequireComponent(typeof(AudioManagerSFX))]
+[RequireComponent(typeof(AudioManagerSkills))]
+
+
+public class AddThisToEverytScriptThatCallsAudio : MonoBehaviour {
```

Comment:
EveryScript (Spelling) and maybe different name?

Reply:
#ArtProgramming